deps/docker-compose.yml

```diff
@@ -14,10 +14,17 @@ services:
     <<: *container-common
     <<: *postgres-common
     image: postgres:13.3-alpine
-    #    command: ["postgres", "-c", "max_connections=100"]
-    command: ["postgres", "-c", "log_statement=all"]
```

Comment:
could you keep this line commented as this is still useful when we are debugging?

Reply:
done